data/chatchannels/scripts/advertising.lua

```diff
@@ -16,8 +16,8 @@ function onSpeak(player, type, message)
 		return true
 	end
 
-	if player:getLevel() == 1 then
-		player:sendCancelMessage("You may not speak into channels as long as you are on level 1.")
+	if player:getLevel() < 20 and not player:isPremium() then
+		player:sendCancelMessage("You may not speak in this channel unless you have reached level 20 or your account has premium status.")
```

Comment:
Is this premium requirement really necessary?

Reply:
It's like a real tibia form and effect players free accounts when config.lua is "freepremium = false"
If anybody like to do free premium, just alter on config.lua to true.

Reply:
Understand. My only point now is that we can merge everything into a single pull. That makes it easier.